app/views/settings/preferences/other/show.html.haml

```diff
@@ -32,6 +35,3 @@
 
   .fields-group
     = f.input :chosen_languages, collection: filterable_languages.sort, wrapper: :with_block_label, include_blank: false, label_method: lambda { |locale| human_locale(locale) }, required: false, as: :check_boxes, collection_wrapper_tag: 'ul', item_wrapper_tag: 'li'
-
-  .actions
-    = f.button :button, t('generic.save_changes'), type: :submit
```

Comment:
Other settings page has many elements too, probably okay to leave button here too.

Reply:
Same here, I put back this button